src/V3AstNodeExpr.h

```diff
@@ -4424,13 +4424,17 @@ class AstSelBit final : public AstNodePreSel {
     // Single bit range extraction, perhaps with non-constant selection or array selection
     // Gets replaced during link with AstArraySel or AstSel
     // @astgen alias op2 := bitp
+private:
+    VAccess m_access;  // Left hand side assignment
```

Comment:
I'm ok with this for now, but noting mostly to myself that long term might be better to have a separate node type for LValue selects.